qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -189,6 +187,11 @@
                     image_snapshot = yes
                     mem = 32768
                     size_mem1 = 32G
+                    # Clearing the private setting of falocate and support xattr
+                    Host_RHEL.m9:
+                        fs_binary_extra_options += " --no-killpriv-v2 --xattr --modcaps=+sys_admin"
+                    Host_RHEL.m8:
+                        fs_binary_extra_options += " -o no-killpriv-v2 -o xattr -o modcaps=+sys_admin"
```

Comment:
Hi @xiagao 
From the codes, I think there are different ways to handle the options between RHEL9 and RHEL8, right?
or it is a kind of special situation？

Reply:
Thanks reminder. I think there is no need this time, I will send another pr to modify all together. Right now it's compatible with two options on rhel9.

Reply:
Okay, it looks good to me.